docs/UserGuide.md

```diff
@@ -283,7 +280,7 @@ Example:
 ### 2.18. Complete or Make Uncomplete a Task : `mark`
 
 Marks a task to be complete or incomplete.
-Format: `mark [complete/uncomplete] INDEX`
+Format: `mark [complete/incomplete] INDEX`
 
 > * Using complete as a parameter will mark the selected task as complete.
 > * Using incomplete as a parameter will mark the selected task as incomplete.
```

Comment:
previously we didn't have a mark command, so it was the only way to mark things complete.
since we have a mark command now, we can either leave it or remove the feature.